daml-lf/interpreter/BUILD.bazel

```diff
@@ -13,7 +13,7 @@ da_scala_library(
     name = "interpreter",
     srcs = glob(["src/main/**/*.scala"]),
     plugins = [
-        "@maven//:com_github_ghik_silencer_plugin_2_12_11",
+        "@maven//:com_github_ghik_silencer_plugin_2_12_12",
```

Comment:
For future reference, #7668 has captured all changes to these silencer strings in scala.bzl.